libraries/lib-track/Track.h

```diff
@@ -1286,6 +1286,15 @@ class TRACK_API TrackList final
    //! Remove all tracks from `list` and put them at the end of `this`
    void Append(TrackList &&list);
 
+   // Like RegisterPendingChangedTrack, but for a list of new tracks,
+   // not a replacement track.  Caller
+   // supplies the list, and there are no updates.
+   // Pending tracks will have an unassigned TrackId.
+   // Pending changed tracks WILL occur in iterations, always after actual
```

Comment:
You copy-pasted an old mistake in the words of the comment.  Should be "new" not "changed"